en/what-do-i-need-to-publish.rst

```diff
@@ -0,0 +1,31 @@
+What do I need to publish?
+=========
+
+Publishing to IATI means you are telling a story about the work your organisation does. For this you break your work down into activities, these could be your programmes or projects. For each activity you detail what the work is, where and when it is happening, who is involved and how it is being financed etc. Further detail can then be added e.g. geographic co-ordinates, results information, document links and policy markers. The IATI technical team recommends that you start by adding the details (in the form of elements) below:
+
+| Topic  | Element | Mandatory | Recommended |
+| --- | --- | :---: | :---: |
+| Identification  | IATI Identifier  | Yes |
+|  | Reporting Org | Yes |
+|Basic Activity Information | Title | Yes |
+|  | Description | Yes |
+|  | Activity Status | Yes |
+|  | Activity Date | Yes |
+| Participating Organisations | Participating Org | Yes |
+| Geopolitical Information | Recipient Country |  | Yes |
+|  | Or, Recipient Region |  | Yes |
+| Classifications | Sector | Yes |
+| Financial | Budget |  | Yes |
+|  | Transactions |  | Yes |
```

Comment:
I’m afraid this table isn’t correctly formatted… This looks very close to being correctly formatted for markdown, but the RST table formatting is a bit different.